site/content/docs/5.0/getting-started/build-tools.md

```diff
@@ -1,14 +1,14 @@
 ---
 layout: docs
 title: Build tools
-description: Learn how to use Bootstrap's included npm scripts to build our documentation, compile source code, run tests, and more.
+description: Contribute to Bootstrap with our documentation build scripts and tests.
 group: getting-started
 toc: true
 ---
 
 ## Tooling setup
 
-Bootstrap uses [npm scripts](https://docs.npmjs.com/misc/scripts/) for its build system. Our [package.json]({{< param repo >}}/blob/v{{< param current_version >}}/package.json) includes convenient methods for working with the framework, including compiling code, running tests, and more.
+Bootstrap uses [npm scripts](https://docs.npmjs.com/misc/scripts/) to build the documentation and compile source files. Our [package.json]({{< param repo >}}/blob/v{{< param current_version >}}/package.json) houses these scripts for compiling code, running tests, and more. These aren't intended for use outside our repository and documentation.
```

Comment:
Wouldn't it be better if this was a warning callout at the top?

Reply:
I hate adding callouts for what can be plainly expressed in other ways. Perhaps we rename this page to `Contribute` or something similar to really reiterate the use case?

Reply:
Yup, agreed, renaming the page sounds better :)